src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1092,6 +1094,20 @@ internal double GetPortVerticalOffset(PortModel portModel)
         /// </summary>
         public void RegisterInputPorts()
```

Comment:
We still use `RegisterInputPorts`, it is not `Obsolete` in fact. We just should make it private/internal. But we can't, because `RegisterInputPorts` is used in many other places.